### PR TITLE
Packages: add Castro

### PR DESCRIPTION
**Pull request by certik**

### CODE REVIEW

File: _data/package_index.yml [hunk 1]
```diff
@@ -608,6 +608,13 @@
   tags: quantum chemistry physics molecular dynamics metadynamics mpi cuda
   license: GNU GPL V2
 
+- name: Castro
+  github: AMReX-Astro/Castro
+  description: An adaptive mesh, astrophysical radiation hydrodynamics simulation code
+  categories: scientific
+  tags: adaptive mesh astrophysics radiation hydrogynamics
```
Comment (LKedward) [inline _data/package_index.yml:615]:
```suggestion
  tags: adaptive mesh astrophysics radiation hydrodynamics
```

File: _data/package_index.yml [hunk 2]
```diff
@@ -608,6 +608,13 @@
...
+  license: BSD 3-Clause
```
Comment (LKedward) [inline _data/package_index.yml:616]:
```suggestion
  license: BSD 3-Clause
  version: none
```

**Comment by certik** [comment]
@LKedward what does `version: none` do? I added the actual version instead.

**Comment by LKedward** [comment]
> @LKedward what does `version: none` do? I added the actual version instead.

It hides the version badge - otherwise it shows an error if it can't find a github version.
I've just realised this repo does have github realises so you don't need the version field, my mistake.

It will look like this:

(<https://img.shields.io/github/v/release/AMReX-Astro/Castro?color=green>)

**Comment by certik** [comment]
@LKedward I see, thanks for the info and the reviews.

**Pull request merged**